frame/staking/src/lib.rs

```diff
@@ -1136,7 +1136,7 @@ decl_module! {
 					}
 				},
 				StakingBalance::KtonBalance(k) => {
-					let stash_balance = T::KtonCurrency::usable_balance(&stash);
+					let stash_balance = T::KtonCurrency::free_balance(&stash);
```

Comment:
Should use usable_balance

Reply:
Okay, I'll modify it back after I finishing the left tests.